modules/qualitycontrol/SendVideoController.js

```diff
@@ -95,27 +83,22 @@ export default class SendVideoController {
      * @private
      */
     _onSenderConstraintsReceived(videoConstraints) {
-        if (FeatureFlags.isSourceNameSignalingEnabled()) {
-            const { maxHeight, sourceName } = videoConstraints;
-            const localVideoTracks = this._conference.getLocalVideoTracks() ?? [];
-
-            for (const track of localVideoTracks) {
-                // Propagate the sender constraint only if it has changed.
-                if (track.getSourceName() === sourceName
-                    && (!this._sourceSenderConstraints.has(sourceName)
-                    || this._sourceSenderConstraints.get(sourceName) !== maxHeight)) {
-                    this._sourceSenderConstraints.set(
-                        sourceName,
-                        maxHeight === -1
-                            ? Math.min(MAX_LOCAL_RESOLUTION, this._preferredSendMaxFrameHeight)
-                            : maxHeight);
-                    logger.debug(`Sender constraints for source:${sourceName} changed to maxHeight:${maxHeight}`);
-                    this._propagateSendMaxFrameHeight(sourceName);
-                }
+        const { maxHeight, sourceName } = videoConstraints;
+        const localVideoTracks = this._conference.getLocalVideoTracks() ?? [];
+
+        for (const track of localVideoTracks) {
+            // Propagate the sender constraint only if it has changed.
+            if (track.getSourceName() === sourceName
+                && (!this._sourceSenderConstraints.has(sourceName)
```

Comment:
Do you really need to check with `has`? It will return `undefined` otherwise, won't it?